var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -1050,6 +1057,8 @@ def cmake_args(self):
                 ]
 
         args += [
+            self.define_from_variant("CP2K_USE_MPI", "mpi"),
+            self.define_from_variant("CP2K_USE_FFTW3", "fftw3"),
```

Comment:
I think we should probably also deal with `CP2K_USE_FFTW3_WITH_MKL`, `CP2K_ENABLE_FFTW3_OPENMP_SUPPORT` and `CP2K_ENABLE_FFTW3_THREADS_SUPPORT` with the `fftw3` variant.